scripts/brp-remove-rpath

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# This script makes sure to remove any standard library search paths from rpath
+# or runpath tags as they're not only redundant, but will break overriding
+# search paths. Search paths of the tag that's not standard will be kept intact.
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+# make sure we have chrpath in our search path, otherwise wuzz out
+type chrpath || exit 0
```

Comment:
You misunderstand the script's purpose.
It doesn't unconditionally remove all rpaths, but rather rpaths for standard search directories such as ie. /usr/lib64 as rpath, which first of all doesn't make sense, secondly it takes presence over $LD_LIBRARY_PATH, breaking the possibility for overriding default search path for binaries with such rpaths.